src/tools/advsec.ts

```diff
@@ -0,0 +1,236 @@
+        case "high":
+          return Confidence.High;
+        case "other":
+          return Confidence.Other;
+        default:
+          return Confidence.Other;
+      }
+    });
+  };
+
+  // Helper function to map string validity to enum values
+  const mapValidity = (validity?: string[]): AlertValidityStatus[] | undefined => {
+    if (!validity) return undefined;
+    return validity.map((v) => {
+      switch (v.toLowerCase()) {
+        case "active":
+          return AlertValidityStatus.Active;
+        case "inactive":
+          return AlertValidityStatus.Inactive;
+        case "none":
+          return AlertValidityStatus.None;
+        case "unknown":
+          return AlertValidityStatus.Unknown;
+        default:
+          return AlertValidityStatus.Unknown;
+      }
+    });
+  };
+
+  server.tool(
+    ADVSEC_TOOLS.get_alerts,
+    "Retrieve Advanced Security alerts for a repository.",
+    {
+      project: z.string().describe("The name or ID of the Azure DevOps project."),
+      repository: z.string().describe("The name or ID of the repository to get alerts for."),
+      alertType: z.enum(["code", "dependency", "secret", "unknown"]).optional().describe("Filter alerts by type. If not specified, returns all alert types."),
+      states: z
+        .array(z.enum(["active", "dismissed", "fixed", "autoDismissed", "unknown"]))
+        .optional()
+        .describe("Filter alerts by state. If not specified, returns alerts in any state."),
+      severities: z
+        .array(z.enum(["critical", "high", "medium", "low", "note", "error", "warning", "undefined"]))
+        .optional()
+        .describe("Filter alerts by severity level. If not specified, returns alerts at any severity."),
+      ruleId: z.string().optional().describe("Filter alerts by rule ID."),
+      ruleName: z.string().optional().describe("Filter alerts by rule name."),
+      toolName: z.string().optional().describe("Filter alerts by tool name."),
+      ref: z.string().optional().describe("Filter alerts by git reference (branch). If not provided and onlyDefaultBranch is true, only includes alerts from default branch."),
+      onlyDefaultBranch: z.boolean().optional().default(true).describe("If true, only return alerts found on the default branch. Defaults to true."),
+      confidenceLevels: z
+        .array(z.enum(["high", "other"]))
+        .optional()
+        .describe("Filter alerts by confidence levels. Only applicable for secret alerts."),
+      validity: z
+        .array(z.enum(["active", "inactive", "none", "unknown"]))
+        .optional()
+        .describe("Filter alerts by validity status. Only applicable for secret alerts."),
+      top: z.number().optional().default(100).describe("Maximum number of alerts to return. Defaults to 100."),
+      orderBy: z.enum(["id", "firstSeen", "lastSeen", "fixedOn", "severity"]).optional().default("id").describe("Order results by specified field. Defaults to 'id'."),
```

Comment:
Can we default order by severity instead? I don't think most customers would sort by id, nor do we by default.